lib/screens/recover_passphrase_entry_screen/recover_passphrase_entry_screen.dart

```diff
@@ -166,11 +154,8 @@ class RecoverPassphraseEntryScreenState
             child: Column(
               key: RecoverPassphraseEntryScreen.wordList,
               mainAxisSize: MainAxisSize.min,
-              children: textControllers.map(
-                (textController) {
-                  final index = textControllers.indexOf(textController);
-                  final focusNode = focusNodes[index];
-
+              children: Iterable<int>.generate(24).toList().map(
```

Comment:
For all the `23` and `24` values, I would base them off of `wordCount` from the bloc